keyboards/mechwild/mercutio/mercutio.c

```diff
@@ -25,6 +24,25 @@ static const char PROGMEM mercutio_name[] = {
     0xC0, 0xC1, 0xC2, 0xC3, 0xC4, 0xC5, 0xC6, 0xC7, 0xC8, 0xC9, 0xCA, 0xCB, 0xCC, 0xCD, 0xCE, 0xCF, 0xD0, 0xD1, 0xD2, 0xD3, 0xD4, 0x00
 };
 
+#ifdef ENCODER_ENABLE
```

Comment:
There is default behavior set in core, now.  Unless you need something different from that, this can be removed.

For refence, it's volume controls if extrakeys are enabled (or of not, if mousekeys, then scroll, otherwise page up/dn).